### PR TITLE
Jeg pinot testing

### PR DESCRIPTION
**Pull request by nitinsingh130519**
Update build.xml to support Apache Pinot release download and on-the-fly building, including Pinot Benchmarks.

This pull request introduces updates to the build.xml file to enhance its functionality. The changes enable the downloading of the Apache Pinot release and building it dynamically during the build process. Additionally, the required Pinot Benchmarks are included.
To validate the effectiveness of these changes, a test run was conducted. The test results can be accessed using the following link: [Test Results](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7824280&view=results).

### CODE REVIEW

File: perf/pinot/build.xml [hunk 1]
```diff
@@ -35,17 +35,20 @@
...
+					<arg line="https://downloads.apache.org/pinot/apache-pinot-0.12.1/apache-pinot-0.12.1-src.tar.gz"/>
 				</exec>
 				<echo message="Extracting Pinot"/>
 				<exec executable="tar" failonerror="true" dir="${DIR_NAME}">
-					<arg line="-xvf apache-pinot-0.12.1-bin.tar.gz"/>
+					<arg line="-xvf apache-pinot-0.12.1-src.tar.gz"/>
 				</exec>
 			</else>
 		</if>
+		<echo message="Building Pinot in ${basedir}/${DIR_NAME}/apache-pinot-0.12.1-src/pinot-perf"/>
```
Comment (karianna) [inline perf/pinot/build.xml:46]: Reason we are building from source as opposed to using the binary?

Comment (nitinsingh130519) [reply perf/pinot/build.xml:46]: @karianna 
Reason for using the source : The provided binaries lack the essential components required for executing the performance benchmarks. They solely include the binaries for running the Pinot application.

Comment (karianna) [reply perf/pinot/build.xml:46]: OK, let's add that as a comment so future you remembers :-)

Comment (nitinsingh130519) [reply perf/pinot/build.xml:46]: @karianna Done.

**Review by karianna — APPROVED** [review]

**Comment by nitinsingh130519** [comment]
[like]  Nitin Singh reacted to your message:
________________________________

**Pull request merged**